README.md

```diff
@@ -74,6 +74,22 @@ See the [oauth2 docs][] for complete instructions on using that library.
 For API methods that require HTTP Basic Authentication, use the
 [`BasicAuthTransport`](https://godoc.org/github.com/google/go-github/github#BasicAuthTransport).
 
+GitHub Apps authentication can be provided by [ghinstallation](https://github.com/bradleyfalzon/ghinstallation).
```

Comment:
Add the word "package" after ghinstallation maybe?

> GitHub Apps authentication can be provided by the [ghinstallation](https://github.com/bradleyfalzon/ghinstallation) package.

Notice how above it says:

> The easiest and recommended way to do this is using the [oauth2](https://github.com/golang/oauth2) **library**, ...